style/themes/lcars.css

```
@@ -1470,6 +1475,15 @@ table.dataTable {
   border-radius: 12px;
 }
 
+.modal-dialog {
+  border-radius: 12px !important;
+}
```

Please, remove the `! important` here:

```suggestion
.modal-dialog {
  border-radius: 12px;
}
```

---

Also, if you are going to add a border to the `.modal-dialog`, please change the color of the borders of the modal-header and modal-footer (the border lines inside the modal are too bright compared to the new background-color):

Color suggestion:

```css
.modal-header,
.modal-footer {
  border-color: #222;
}
```